README.md

```diff
@@ -19,7 +19,7 @@ Binaries are available [in the releases section](https://github.com/stefansundin
 ## Clone Repository
 
 ```bash
-$ git clone https://github.com/stefansundin/powermate-linux.git
+$ git clone --recursive-submodules https://github.com/stefansundin/powermate-linux.git
```

Comment:
Typo here, this should be `--recurse-submodules`.

Reply:
Ouch! I've even looked through my shell history to make the change, but I didn't review my changes before making the pull request, sorry!

I've pushed a new change that changes things to the correct command. That pull request will show up soon and I'll close this one.

Reply:
Actually, the PR seems to automatically track my force-pushes, so I'll keep things within this PR.